js-wether/wether.js

```diff
@@ -0,0 +1,96 @@
+document.addEventListener("DOMContentLoaded", function () {
+    //指定されたプルダウンの値を取得
+    let getSelectValue = function (name) {
+        let result = "";
+        let elems = document.getElementById(name).options;
+        for (let i = 0, len = elems.length; i < len; i++) {
+            let elem = elems.item(i);
+            if (elem.selected) {
+                result = elem.value;
+                break;
+            }
+        }
+        return result;
```

Comment:
F.Y.I

`querySelector`や`querySelectorAll`というメソッドがあります。
これらを使うと、スッキリかけます。

修正は不要です。

```suggestion
  const result =  document.getElementById("city").querySelector('option:checked').value;
   // => "New York"
```

- 必ず選択済みの要素がある前提で記述しています。